pom.xml

```diff
@@ -611,10 +616,15 @@
         <finalName>${project.artifactId}-${project.version}</finalName>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-enforcer-plugin</artifactId>
+                    <version>3.3.0</version>
+                </plugin>
```

Comment:
We've lived very long without this, so I'd like to know why this would be needed / beneficial. Otherwise, I'd rather not include new plugins.